src/components/Widget/index.js

```diff
@@ -419,6 +432,8 @@ class Widget extends Component {
               }
             }
           }
+          // Update the stored item in sessionStorage
+          sessionStorage.setItem(SESSION_NAME, JSON.stringify(object));
```

Comment:
Why would you set the modified `object` with no message back to the `sessionStorage`?